### DISCUSSION TITLE
How do I get the color palette from the tailwind docs to work?

### DISCUSSION THREAD

**Discussion by dmill** [rails/tailwindcss-rails#323] [General]
Hello,

I'm working on a fresh rails 7 app with `-c tailwind`. I am unable to use the default colors documented [here](https://tailwindcss.com/docs/customizing-colors#using-the-default-colors). I've been googling around for this for awhile and nothing seems to work to get these colors to load. They work just find with a fresh react/next setup. Can anyone advise as to how to get these working in Rails without manually defining each color in `tailwind.config.js`?

**Comment by dmill** [comment]
For example, `bg-blue-500` or `bg-gray-100` will both work, `bg-red-500` and `bg-red-100` will not.

**Comment by flavorjones** [comment]
Hi @dmill, can you help me understand what you're seeing or experiencing when you say "I am unable to use the default colors"?

- Are you getting an error message, or do you not see what you expect in the web page?
- Can you share what changes you made to your config and view files?
- Can you also share what commands you're running, and what the output is that you see?

Here's what works for me:

- `rails new`
- `bundle add tailwindcss-rails`
- `bin/rails tailwindcss:install`
- `rails generate scaffold post title:string body:text`
- run `bin/dev` (or use the puma plugin, or run `bin/rails tailwindcss:watch` -- see the README)
- browse to http://127.0.0.1:3000/posts/new
- edit `app/views/posts/_form.html.erb` to change the button color from `bg-blue-600` to `bg-pink-600`
- refresh the page and see a pink button

What are you doing differently from me?

**Comment by modagavr** [comment]
@dmill 
Seems like you precompiled assets and tailwindcss built your css with only used classes.

To fix, delete `public/assets` folder and tailwind will again launch in dev mode.

Also don't forget to both launch rails server and tailwindcss:watch as @flavorjones mentioned

**Comment by dmill** [comment]
Thanks guys, I believe this was related to accidentally precompiled assets. No idea how I managed to do that, must've been an errant copy/paste that included it that I didn't realize. Thanks!
